src/rt/minfo.d

```diff
@@ -339,8 +353,7 @@ struct ModuleGroup
                 else
                 {
                     auto midx = edges[sp.curMod][sp.curDep];
-                    // if midx is -1, then this isn't part of this DSO.
-                    if (midx != -1 && !bts(reachable, midx))
+                    if (!bts(reachable, midx))
```

Comment:
> That means we could change the type of edges to uint, right?

Hm... probably could. I don't think it's too important though, no chance of having over 2 billion modules to sort. I'm leery of unsigned math, too.